apps/etterlatte-saksbehandling-ui/client/src/components/person/brev/NyttBrevModal.tsx

```diff
@@ -60,50 +58,46 @@ export const NyttBrevModal = ({ sakId }: { sakId: number }) => {
           </Heading>
         </Modal.Header>
 
-        <form onSubmit={handleSubmit((data) => lagre(data))}>
+        <form onSubmit={handleSubmit((data) => opprettBrev(data))}>
```

Comment:
```suggestion
        <form onSubmit={handleSubmit(opprettBrev)}>
```
burde være det samme?